Open Judge System/LocalWorker/OJS.LocalWorker/OjsSubmissionProcessingStrategy.cs

```diff
@@ -166,7 +165,16 @@ protected override void ProcessTestsExecutionResult(IExecutionResult<TestResult>
 
             this.UpdateResults();
         }
-
+        
+        private static void UpdateParticipantTotalScore(Participant participant)
+        {
+            participant.TotalScoreSnapshot =
+                participant.Scores.Where(ps => !ps.Problem.IsDeleted)
+                    .Select(ps => ps.Points)
+                    .Sum();
```

Comment:
This again will make db request to retrieve the Scores.

Reply:
Done